docusaurus/docs/dev-docs/migration/v4-to-v5/additional-resources/plugins-migration.md

```diff
@@ -1,35 +1,50 @@
 ---
-title: Plugins upgrade summary
+title: Plugins upgrade
 displayed_sidebar: devDocsMigrationV5Sidebar
 tags:
   - plugins
   - plugins development
   - upgrade to Strapi 5
 ---
 
-# Plugins upgrade summary
+# Plugins upgrade
 
 The present page is intended to be used as a short summary of everything to consider if you are a plugin developer intending to upgrade your plugin from Strapi v4 to Strapi 5. The page quickly describes the changes affecting plugins and links to additional resources where necessary.
 
+:::caution
```

Comment:
```suggestion
:::strapi
```

Downgrading to a "strapi" (the purple, special) callout as caution are for risks (losing data, etc.)